packages/examples/aws-codebuild/src/codebuild_test.sh

```diff
@@ -4,6 +4,8 @@ set -euxo pipefail
 echo "Run dockest as usual"
 yarn test:codebuild:buildspec
 
-# image source: https://github.com/n1ru4l/docker-image-node-10-with-docker-and-compose
+echo "Build docker image"
+docker build --file CI.Dockerfile -t aws-codebuild-ci-node .
```

Comment:
As mentioned above we now use the image inside this repo :) I do not intend to maintain the other one as we are using a custom one at work anyways. So we can maintain a version co-located with dockest that can be adjusted when tests fail/the image becomes outdated.